src/select.rs

```diff
@@ -87,3 +92,40 @@ impl<'a> TElement<'a> for &'a ElementData {
         }
     }
 }
+
+impl<'a> Node<'a> {
+    pub fn css(&'a self, css_str: &str) -> FilterNodes<'a, Descendants<'a>> {
+        let selectors = match parser::parse_author_origin_selector_list_from_str(css_str) {
+            Ok(vec) => vec,
+            Err(_) => Vec::new(),
```

Comment:
Errors should not pass silently. This method should return a `Result` and return `Err` when given an invalid selector.